src/System.Text.Json/src/System/Text/Json/JsonTokenType.cs

```diff
@@ -15,17 +15,68 @@ public enum JsonTokenType : byte
     {
         // Do not re-order.
         // We rely on the ordering to quickly check things like IsTokenTypePrimitive
+
+        /// <summary>
+        ///   Indicates that there is no value (as distinct from <see cref="Null"/>).
+        /// </summary>
+        /// <remarks>
+        ///   This is the default token type if no data has been read by the <see cref="Utf8JsonReader"/>.
```

Comment:
I suggest you use the full DocId, like this:

```suggestion
        ///   This is the default token type if no data has been read by the <see cref="T:System.Text.Json.Utf8JsonReader"/>.
```